lib/iris/plot.py

```diff
@@ -260,7 +281,12 @@ def _draw_2d_from_bounds(draw_method_name, cube, *args, **kwargs):
         for coord, axis_name, data_dim in zip([u_coord, v_coord],
                                               ['xaxis', 'yaxis'],
                                               [1, 0]):
-            if coord:
+            if coord is None:
```

Comment:
Too late. Just seen it was the `else` below.
I think we can do better to preserve the existing logic and still support `dim=0` :wink